### PR TITLE
Teleinfo

### PR DESCRIPTION
**Pull request by 0hax**
## Description:

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#1108

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Review by jesserockz — APPROVED** [review]
LGTM.

File: components/sensor/teleinfo.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
+Teleinformation from Linky electrical counter
+=============================================
+
+.. seo::
+    :description: Instructions for setting up French Teleinformation
+    :image: mhz19.jpg
```
Comment (jesserockz) [inline components/sensor/teleinfo.rst]: This looks like the wrong image used?

```suggestion
    :image: teleinfo-full.jpg
```

or

```suggestion
    :image: teleinfo.jpg
```

**Review by jesserockz — CHANGES_REQUESTED** [review]
Just 1 minor change plus deleting the images that are not actually used please.

**Comment by 0hax** [comment]
> Just 1 minor change plus deleting the images that are not actually used please.

I removed the image and updated the branch.
Thanks

File: components/sensor/teleinfo.rst (hunk 1 above)
Comment (jesserockz) [reply components/sensor/teleinfo.rst]: @0hax maybe you missed this? which of the images should be used for SEO?

**Review by jesserockz — APPROVED** [review]

**Pull request merged**